Mathlib/Algebra/Category/Ring/FilteredColimits.lean

```diff
@@ -179,23 +180,27 @@ def colimit : CommSemiRingCat.{max v u} :=
 def colimitCocone : Cocone F where
   pt := colimit.{v, u} F
   ι :=
-    { (SemiRingCat.FilteredColimits.colimitCocone
-          (F ⋙ forget₂ CommSemiRingCat SemiRingCat.{max v u})).ι with }
+    { app := fun X ↦ ofHom <| ((SemiRingCat.FilteredColimits.colimitCocone
+          (F ⋙ forget₂ CommSemiRingCat SemiRingCat.{max v u})).ι.app X).hom
+      naturality := fun _ _ f ↦ hom_ext <|
+        RingHom.coe_inj ((Types.TypeMax.colimitCocone
+          (F ⋙ forget CommSemiRingCat)).ι.naturality f) }
```

Comment:
Hmm, this seems unfortunate.

Reply:
Can we provide a function for this?

Reply:
I am afraid, I don't understand. This worked before, because it abused the def-eq of morphisms in `CommSemiRingCat` and `SemiRingCat`. Do you suggest to add some general construction for `(C : Type*) [Category C] [ConcreteCategory C] [HasForget_2 C SemiRingCat]`?

Reply:
Yes, I guess so. I was imagining something that can take a natural transformation, unwrap it and apply `hom` to the morphism, and `hom_ext` to the naturality condition, and put it back together. But I didn't think to hard about whether there is a type signature that makes sense. :-)